NODERS/chains.json

```diff
@@ -325,6 +325,15 @@
         "run_time": "every 1 hour",
         "minimum_reward": 1000000
       }
+    },
+    {
+      "name": "arkeo",
+      "address": "tarkeovaloper1hsrprv6xpfqrn2x46c4v0jxxvp4hhwrdqw483f",
```

Comment:
Is this the right prefix? I'm not familiar with the chain but usually validator and account address prefix should match. I'll merge but submit another update if needed

Reply:
yep, seems like that an error, copied valoper from explorer without checking